core/platform/email/gae_email_services.py

```diff
@@ -22,6 +22,15 @@
 
 
 def get_incoming_email_address(reply_to_id):
+    """Returns string detailing the id of the reply-client.
+
+    Args:
+      - reply_to_id: str or None. The unique reply-to id used in reply-to email
+          sent to recipient.
+
+    Returns:
+      A str detailing the id of the reply-client.
```

Comment:
Make it look simial to this :+1: https://github.com/oppia/oppia/blob/ac42e7fd96de1faf610634f2a7f495c4afa15bc1/core/domain/classifier_domain.py#L131-L134

i.e, add return type first then the description.

Reply:
Also, indent it by 4.

Reply:
Done